assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -927,6 +927,11 @@ form {
   background-color: #ededed;
 }
 
+.frequencies__placeholder .form__radio-group-label {
+  background-color: #ffffff;
```

Comment:
You'll wanna use `gu-colour(neutral-100)`

Reply:
thanks, did that now! 👍